Scripts~/windows/.export-server-sdk.ps1

```diff
@@ -0,0 +1,49 @@
+param(
+	[Parameter(Mandatory)]
+	[Alias("ServerSdk", "SdkVersion", "Sdk")]
+	[ValidatePattern("\d+\.\d+\.\d+")]
+	[string] $ServerSdkVersion
+)
+
+$ROOT_DIR='.'
+$SERVER_SDK_FILENAME="GameLift-CSharp-ServerSDK-UnityPlugin-$ServerSdkVersion.zip";
+$DESTINATION_PATH="$ROOT_DIR\$SERVER_SDK_FILENAME"
+$TEMP_PATH="C:\Temp"
+$TEMP_ZIP_PATH = "$TEMP_PATH\GameLiftServerSDK.zip"
```

Comment:
Why not have this be "$TEMP_PATH\$SERVER_SDK_FILENAME"? Why rename it to "GameLiftServerSDK.zip"?

Reply:
My guess is there was concern about filepath sizing in Windows; however, when looking, the longest path we could hit was about 150 characters with the new directory and temp filenames which gives us ~100 character buffer. Updated per your suggestion